nature/models.py

```diff
@@ -29,10 +28,19 @@ def for_admin(self):
         return self
 
     def for_office_hki(self):
-        return self.filter(protection_level__gte=PROTECTION_LEVELS['OFFICE_HKI'])
+        """For office users who work for City of Helsinki"""
+        return self.filter(protection_level__gte=PROTECTION_LEVELS['OFFICE'])
 
     def for_office(self):
-        return self.filter(protection_level__gte=PROTECTION_LEVELS['OFFICE'])
+        """For office users who do not work for City of Helsinki
+
+        For models protected by protection levels but not related
+        to Feature or FeatureClass model, for_office_hki and for_office
+        returns the exact same queryset. But for Feature or FeatureClass
+        related models, the queryset should be furthermore filtered by
+        excluding UHEX features
+        """
+        return self.for_office_hki()
```

Comment:
I don't get this, seems this returns the same office queryset for both `for_office` and `for_office_hki`?

Reply:
Thanks for the explanation, I had forgotten about this and would have asked as well, but I get it now :) I would say the main benefit of it is working as a interface for the class right here.